plugins/@grouparoo/airtable/package.json

```diff
@@ -31,7 +31,6 @@
   "dependencies": {
     "@grouparoo/app-templates": "0.8.0-alpha.5",
     "@grouparoo/core": "0.8.0-alpha.5",
-    "actionhero": "28.1.2",
```

Comment:
Was this change on purpose?

Reply:
Yes, since it was in both deps and dev deps I thought that was an accident.